distribution/solr-distro/assembly.xml

```diff
@@ -36,39 +36,19 @@
                     org.locationtech.jts:jts-core
                 </include>
             </includes>
-            <outputDirectory>/server/lib</outputDirectory>
+            <outputDirectory>/server/solr-webapp/webapp/WEB-INF/lib</outputDirectory>
         </dependencySet>
         <dependencySet>
             <includes>
                 <include>org.codice.pro-grade:pro-grade</include>
             </includes>
             <outputDirectory>/security</outputDirectory>
         </dependencySet>
-        <dependencySet>
```

Comment:
Removed all of the special handling for these jackson jars since Solr 7.7.2 depends on jackson version 2.9.8 instead of 2.9.5.